data/start.txt

```diff
@@ -36,7 +36,7 @@ conversation "default intro"
 	`The bank's architecture is what you would have called "futuristic" back when you dreamed that the future would be less squalid than the present: story above story of curves and spires and balconies, all in gleaming metal. A doorman stands by each of the heavy glass doors. You are wearing your very best clothes, but you feel shabby next to them.`
 	`	The loan broker's office is equally intimidating. Large computer monitors fill the walls, with stock quotes and other information spilling down them almost too fast to read. You occasionally glimpse the names of places that you have only seen in movies: Aldebaran. Tarazed. Earth.`
 	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, flips the pages past you quickly, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
-	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 434,000 credits in interest. Sign here." He grins.`
+	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,577 credits in interest. Sign here." He grins.`
```

Comment:
As long as we're trying to be correct, where did you get this number? I ran the code for paying for the given loan over the course of its entire term while not ever paying any extra and got a total payment of 913,567, which is 433,567 paid in interest, not 433,577.

Reply:
I got this via the attached spreadsheet. As this is not an integer and there are no fractional credits I can either round down or up. I rounded to the nearest integer in my comment, but I rounded up for the banker's comment.
[Endless Sky Initial Loan.xlsx](https://github.com/endless-sky/endless-sky/files/11070373/Endless.Sky.Initial.Loan.xlsx)